Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
Can't be more "light weight" than this, as it's already built-in in the framework ;)

Here are my 2 cents, so that we can reach @sophiedeziel's dollar some day! Light weight is certainly good from a design point of view :+1: But I want to emphasize a small detail. Due to the implementation of `to_param`, when it comes to routing, anything after the dash (`-`) in the url will be ignored. For example, if you have a page model with `id` = 1, then `/pages/1-ignored-garbage-characters` would be a perfectly routable url, as the id (1 in this case) is what counts. From Google's point of view, that means it's theoretically possible to create pages with duplicate content, which is bad SEO. Also, having the `id` in the url is also not optimal.

Bottom line: my guess is that SEO is not that important for that project, so I would also be in favor of using `to_param`, at least until the day this project needs a tad more sophisticated url slugs. 

That said, it's now my turn to shut up :smile_cat: